### PR TITLE
MAR integration

### PR DESCRIPTION
**Pull request by asafshen**
fixes: https://github.com/demisto/server/issues/7570

Also sent a different version to @gshriki (works at current build version - because we use var from common)

### CODE REVIEW

**Comment by asafshen** [comment]
@meirwah thanks,

Regarding the dbot score - MAR doesn't return any score upon `file`/`ip`

**Comment by meirwah** [comment]
In that case @asafshen maybe it's not a good idea to call them !ip/!url/!file commands, not sure..
@liorkol @Spivaka  WDYT?

File: Integrations/integration-McAfee-MAR.yml [hunk 1]
```diff
@@ -0,0 +1,462 @@
+commonfields:
+  id: McAfee Active Response
+  version: -1
+name: McAfee Active Response
+display: McAfee Active Response
+category: Authentication
```
[the remark this answers is not in the document]
Comment (NeelRust) [reply Integrations/integration-McAfee-MAR.yml]: Category should be `Endpoint`. One word.

**Comment by aviadl** [comment]
@meirwah is right, if commands do not have reputation, they should not use the standard reputation commands `ip` , `file` etc

**Comment by asafshen** [comment]
Thanks. Removed`!ip` & `!file`  & change category

File: Integrations/integration-McAfee-MAR.yml [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+      - Software
+      - Startup
+      - UsbConnectedStorageDevices
+      - UserProfiles
+      - WinRegistry
+      description: The collector to query
+    - name: outputs
+      description: Comma separated fields (outputs) to view - run `mar-collectors-list`
+        command for all collectors outputs, shows all outputs if not supplied
+      isArray: true
```
Comment (meirwah) [inline Integrations/integration-McAfee-MAR.yml]: are you sure it should be array??
cause you ask here for  Comma separated ....

File: Integrations/integration-McAfee-MAR.yml [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+  required: true
+- display: Client certificates content (see `client.crt` in instructions)
+  name: cert_file
+  defaultvalue: ""
+  type: 12
+  required: true
+- display: Client private key content (see `client.key` in instructions)
+  name: private_key
+  defaultvalue: ""
+  type: 12
```
Comment (meirwah) [inline Integrations/integration-McAfee-MAR.yml]: you may want to change this to type 14
Which is ConfigDataTypeTextAreaEncrypted

File: Integrations/integration-McAfee-MAR.yml [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+
+    def search_result_to_table(search_result):
+        items = search_result['items']
+        table = [extract_item_output(item) for item in items]
+        return table
+
+    def search(collector, outputs, filter_by, filter_operator, filter_value):
+        config = get_client_config()
+        with DxlClient(config) as client:
+            x = client.connect()
```
Comment (Spivaka) [inline Integrations/integration-McAfee-MAR.yml]: x??

**Comment by asafshen** [comment]
Waiting for our MAR instance to enabled again so we can add the playbook test

File: Integrations/integration-McAfee-MAR.yml (hunk 4 above)
Comment (asafshen) [reply Integrations/integration-McAfee-MAR.yml]: x mark the spot

**Pull request merged**